config/initializers/lograge.rb

```diff
@@ -11,6 +11,9 @@
   # Log to the same place as Rails logs
   config.lograge.logger = Rails.logger
 
+  # Let's not log the healthcheck (and maybe other things, eventually)
+  config.lograge.ignore_actions = ['PublicPagesController#healthcheck']
```

Comment:
These events can be valuable, especially when investigating networking issues between the load balancer and the application.

Could we add an environment variable that we can use to enable logging if needed? Something like:

```suggestion
  config.lograge.ignore_actions = ['PublicPagesController#healthcheck'] unless ENV.fetch('ENABLE_HEALTHCHECK_LOGGING', false)
```